content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Operator pattern
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Operators are software extensions to Kubernetes that make use of third-party
+resources to manage applications and their components. Operators follow
+Kubernetes principles, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Motivation
+
+The Operator pattern aims to capture the key aim of a human operator who
+is managing a service or set of services. Human operators who look after
+specific applications and services have deep knowledge of how the system
+ought to behave, how to deploy it, and how to react if there are problems.
+
+People who run workloads on Kubernetes often like to use automation to take
+care of repeatable tasks. The Operator pattern captures how you can write
+code to automate a task beyond what Kubernetes itself provides.
+
+## Operators in Kubernetes
+
+Kubernetes is designed for automation. Out of the box, you get lots of
+built-in automation from the core of Kubernetes. You can use Kubernetes
+to automate deploying and running workloads, *and* you can automate how
+Kubernetes does that.
+
+Kubernetes' {{< glossary_tooltip text="controllers" term_id="controller" >}}
+concept lets you extend the cluster's behaviour without modifying the code
+of Kubernetes itself.
+Operators are clients of the Kubernetes API that act as controllers for
+a [Custom Resource](/docs/concepts/api-extension/custom-resources/).
+
+## An example Operator {#example}
+
+Some of the things that you can use an operator to automate include:
+
+* deploying an application on demand
+* taking and restoring backups of that application's state
+* handling upgrades of the application code alongside related changes such
+  as database schemas or extra configuration settings
+* publishing a Service to applications that don't support Kubernetes APIs to
+  discover them
+* simulating failure in all or part of your cluster to test its resilience
+* choosing a leader for a distributed application without an internal
+  member election process
+
+What might an Operator look like in more detail? Here's an example in more
+detail:
+
+1. a custom resource named SampleDB, that you can configure into the cluster
+2. a Deployment that makes sure a Pod is running that contains the
+   Controller part of the operator
```

Comment:
The style of 1 and 2 here should be changed to match the rest of the example, start with a capital, end with a full stop.

Reply:
Thanks. Tweaked.